Packages/Testing-MIES/UTF_BasicHardwareTests.ipf

```diff
@@ -1862,6 +1862,34 @@ Function StartDAQDuringTP_IGNORE(device)
 	wv[%$"Analysis function (generic)"][%Set] = "WriteIntoLBNOnPreDAQ"
 End
 
+Window AbortTP_dummy() : Panel
+	PauseUpdate; Silent 1		// building window...
+	NewPanel /W=(150,77,450,277) as "AbortTP_dummy"
+	SetWindow kwTopWin,userdata(random_change)=  "438.25"
+EndMacro
+
+// UTF_TD_GENERATOR HardwareMain#DeviceNameGeneratorMD1
+Function AbortTP([str])
+	string str
+
+	STRUCT DAQSettings s
+	InitDAQSettingsFromString(s, "MD1_RA0_I0_L0_BKG_1_RES_0")
+	AcquireData(s, str, startTPInstead=1)
+
+	CtrlNamedBackGround DelayReentry, start=(ticks + 300), period=60, proc=JustDelay_IGNORE
+	RegisterUTFMonitor("DelayReentry", BACKGROUNDMONMODE_AND, "AbortTP_REENTRY", timeout = 600, failOnTimeout = 1)
+End
+
+Function AbortTP_REENTRY([str])
+	string str
+
+	string device
+
+	device = StringFromList(0, str)
+	KillWindow $device
+	ASYNC_Stop(timeout = 5)
```

Comment:
How about calling ASYNC_START here as in

```igorpro
try
 ASYNC_STOP(timeout = 5)
 aborted = 0
catch
 err = getRTError(1)
  aborted = 1
endtry

ASYNC_Start(threadprocessorCount, disableTask=1)

if(aborted)
  FAIL()
else
  PASS()
endif
```

as that does not influence the other test cases?

Reply:
ok, I fixed the commit.